misc/README.md

```diff
@@ -0,0 +1,3 @@
+# wt024.ttf
+
+The file was copyied from https://github.com/dictcp/wangfonts/blob/master/TrueType/wt024.ttf and is used in benches only.
```

Comment:
"copied". It should probably mention that the file is released under GNU GPL 2. I'm slightly uneasy including GPL artifacts in this repo, but given that it is only used for benchmarking I think it's ok. If it ever becomes a problem we can replace it.

Reply:
Fixed